syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -1,39 +1,54 @@
-import random
-from operator import add, sub
+from operator import add, sub, mul
 import torch
 import syft
 from syft.generic.abstract.tensor import AbstractTensor
+from syft.frameworks.torch.mpc.przs import PRZS, gen_alpha_3of3
 
 
 class ReplicatedSharingTensor(AbstractTensor):
     def __init__(
-        self, shares_map=None, owner=None, id=None, tags=None, description=None,
+        self, plain_text=None, players=None, owner=None, id=None, tags=None, description=None,
     ):
         super().__init__(id=id, owner=owner, tags=tags, description=description)
+        self.ring_size = 2 ** 32
+        shares_map = self.__generate_shares_map(plain_text, players)
```

Comment:
Q: Shouldn't we do this only if we have ```plain_text != None```?
(maybe we can have two ways in which we construct RSS, one when we send the plaintext to the constructor and one using the ```share``` method)

Reply:
this is checked in the __generate_shares_map function. if plain_text's None it returns None

Reply:
Hmm...seems ok - when refactoring we should stick with the same name method (we have ```share``` for ```AST``` and we do the sharing in the constructor for the ```RST```)